test/unit/org/apache/cassandra/inject/ActionBuilder.java

```diff
@@ -35,7 +33,7 @@
  */
 public class ActionBuilder
 {
-    private Class<?> helperClass = InjectorHelper.class;
```

Comment:
@mike-tr-adamson do you know why do we need `InjectorHelper`?

Reply:
TBH, I don't know why we need it. It was part of the original code. I was going to try removing it and seeing if the tests still worked.